drivers/mge-hid.c

```diff
@@ -1482,8 +1484,9 @@ static hid_info_t mge_hid2nut[] =
 	{ "outlet.1.id", 0, 0, "UPS.OutletSystem.Outlet.[2].OutletID", NULL, "%.0f", HU_FLAG_STATIC, NULL },
 	{ "outlet.1.desc", ST_FLAG_RW | ST_FLAG_STRING, 20, "UPS.OutletSystem.Outlet.[2].OutletID", NULL, "PowerShare Outlet 1", HU_FLAG_ABSENT, NULL },
 	{ "outlet.1.switchable", 0, 0, "UPS.OutletSystem.Outlet.[2].PresentStatus.Switchable", NULL, "%s", HU_FLAG_STATIC, yes_no_info },
-	{ "outlet.1.status", 0, 0, "UPS.OutletSystem.Outlet.[2].PresentStatus.SwitchOn/Off", NULL, "%s", 0, on_off_info },
+	{ "outlet.1.status", 0, 0, "UPS.OutletSystem.Outlet.[1].Status", NULL, "%s", 0, on_off_info },	
```

Comment:
ditto, may need both.... this one provides more than just on/off

Global status of the load segment :
0 : Not powered
1 : Not protected
2 : Protected

Reply:
so to leave { "outlet.2.status", 0, 0, "UPS.OutletSystem.Outlet.[3].PresentStatus.SwitchOn/Off", NULL, "%s", 0, on_off_info }, ?

Reply:
0,1,2 thius mean =  NULL, "%.2f", 0, NULL }, or  NULL, "%.0f", HU_FLAG_STATIC, NULL },

Reply:
means a lookup function, something like:

static info_lkp_t eaton_enable_disable_info[] = {
	{ 0, "not powered", NULL, NULL },
	{ 1, "not protected", NULL, NULL },
	{ 2, "protected", NULL, NULL },
	{ 0, NULL, NULL, NULL }
};